jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Button.java

```diff
@@ -0,0 +1,27 @@
+package com.epam.jdi.light.material.elements.inputs;
+
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.material.asserts.inputs.ButtonAssert;
+
+public class Button extends UIBaseElement<ButtonAssert> {
+
+    @JDIAction("Click '{name}'")
+    public void click() {
+        core().click();
+    }
+
+    @Override
+    public ButtonAssert is() {
+        return new ButtonAssert().set(this);
+    }
+
+    @Override
+    public ButtonAssert has() {
```

Comment:
you can call is() inside of has() to avoid code duplications

Reply:
fixed